templates/database_cleaner_rspec.rb

```diff
@@ -1,21 +1,28 @@
 RSpec.configure do |config|
+  config.use_transactional_fixtures = false
```

Comment:
If the file name is `database_cleaner`, it feels unexpected to me to have this here. Feels better grouped next to the `ActiveRecord::Migration.maintain_test_schema!` line.

Could rename the file `database.rb` or something